presto-hive/src/main/java/io/prestosql/plugin/hive/azure/HiveAzureConfig.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.hive.azure;
+
+import io.airlift.configuration.Config;
+import io.airlift.configuration.ConfigSecuritySensitive;
+
+import javax.annotation.PostConstruct;
+
+import java.util.Optional;
+
+import static com.google.common.base.Preconditions.checkState;
+import static com.google.common.base.Strings.isNullOrEmpty;
+
+public class HiveAzureConfig
+{
+    private String wasbStorageAccount;
+    private String wasbAccessKey;
+
+    @PostConstruct
+    public void validate()
```

Comment:
i think this should be `javax.validation`. Let's 
https://github.com/prestosql/presto/blob/6d4272abaa2446ae849ab7c940adb1f20a4ab2a4/presto-main/src/main/java/io/prestosql/sql/analyzer/FeaturesConfig.java#L669-L673

Reply:
or do the check in `PrestoAzureConfigurationInitializer` ctor